src/openrct2/actions/PlacePeepSpawnAction.cpp

```diff
@@ -46,8 +46,9 @@ GameActions::Result PlacePeepSpawnAction::Query() const
     res.Expenditure = ExpenditureType::LandPurchase;
     res.Position = _location;
 
-    if (!LocationValid(_location) || _location.x <= 16 || _location.y <= 16 || _location.x >= (GetMapSizeUnits() - 16)
-        || _location.y >= (GetMapSizeUnits() - 16))
+    auto mapSizeUnits = GetMapSizeUnits() - CoordsXY{ 16, 16 };
```

Comment:
We should have a constant for 16 too, I think. Something like `HALF_TILE`. But this can be done in another PR too.